src/components/Question.js

```diff
@@ -25,7 +27,8 @@ const Collapsed = ({ setExpanded }) => {
     replies[replies.length - 1]
   const replyCount = replies.length - 2
   const maxAvatars = Math.min(replyCount, 3)
-  const replyAuthorMetadata = reply?.profile?.metadata[0]
+  const replyAuthorMetadata =
+    reply?.profile?.profiles_readonly?.[0] || reply?.profile?.metadata?.[0]
```

Comment:
@smallbrownbike this is a property name change, and an attempt to support both for the moment, though we should be able to remove the `metadata` branch of the conditional after we deploy this.